### ISSUE TITLE
[cssom] Publish an updated WD

### ISSUE THREAD

**Issue by xfq** [w3c/csswg-drafts#6446]
There have been [substantive changes](https://github.com/w3c/csswg-drafts/commits/main/cssom-1) and the last publication was in 2016. Although there are still a lot of open issues, I suggest that we publish an updated WD.

I sent a PR to update the Changes section: https://github.com/w3c/csswg-drafts/pull/6445

**Comment by svgeesus** [comment]
@emilio

**Comment by emilio** [comment]
Yes, that seems worth it. Do I need to put this on the agenda or do working drafts not need a resolution?

**Comment by svgeesus** [comment]
It needs a quick agenda item.

**Comment by svgeesus** [comment]
LGTM, publication requested 24 Aug expected 26 Aug
